va/va.go

```diff
@@ -232,8 +247,9 @@ func (va VAImpl) process(task *vaTask) {
 
 func (va VAImpl) performValidation(task *vaTask, results chan<- *core.ValidationRecord) {
 	if va.sleep {
-		// Sleep for a random amount of time between 1-15s
-		len := time.Duration(rand.Intn(15))
+		// Sleep for a random amount of time between 1-15s (where 15 can be
```

Comment:
I think you can delete this comment outright. It's more confusing now with when the `va.sleepTime` might not always be 15s and I think the code is clear enough without the comment.

Reply:
I changed it to `// Sleep for a random amount of time between 0 and va.sleepTime seconds`